src/commands/mobile_app/upload.rs

```diff
@@ -0,0 +1,26 @@
+use anyhow::Result;
+use clap::ArgAction;
+use clap::{Arg, ArgMatches, Command};
+
+use crate::utils::args::ArgExt;
+
+pub fn make_command(command: Command) -> Command {
+    command
+        .about("Upload mobile app files to a project.")
+        .org_arg()
+        .project_arg(false)
+        .arg(
+            Arg::new("paths")
+                .value_name("PATH")
+                .help("The path to the mobile app files to upload. Supported files include Apk, Aab or XCArchive.")
+                .num_args(1..)
+                .action(ArgAction::Append),
+        )
+}
+
+#[allow(clippy::unnecessary_wraps)]
```

Comment:
Please use `expect` rather than `allow`.

The difference is that the linter will complain if you use an `expect` where it is not needed, whereas there is nothing to indicate if you have an unnecessary `allow`. So, once this function starts returning an error `clippy` will remind you to remove this if you use `expect`.

```suggestion
#[expect(clippy::unnecessary_wraps)]
```